src/components/chat/render-file-upload-message.jsx

```
@@ -1,6 +1,6 @@
 import PropTypes from "prop-types";
 import CustomAudioPlayer from "./custom-audio-player";
-import { FileIcon } from "lucide-react";
+import { FileIcon, ImageIcon, AudioWaveformIcon } from "lucide-react";
```

ESLint

[error] 'ImageIcon' is defined but never used.

(no-unused-vars)